src/org/etools/j1939_84/controllers/part11/Part11Step13Controller.java

```diff
@@ -644,108 +658,135 @@ private void testSp12675(OBDModuleInformation module) {
+                            :
+                            String.format("0x%08X", checkValue);
                     if (spn.getRawValue() > checkValue) {
                         // 6.11.13.4.c. Fail each PG query where any bin value received is greater than FAFFh. (Use
                         // FAFFFFFFh for NOx values)
-                        addFailure("6.11.13.4.c - Bin value received is greater than " + checkValueString + "(h) from "
-                                + module.getModuleName() + " for " + spn);
+                        addFailure(
+                                "6.11.13.4.c - Bin value received is greater than " + checkValueString + "(h) from "
+                                        + module.getModuleName() + " for " + spn);
 
                     }
                     // 6.11.13.4.d. Fail all values where the corresponding value received in part 2 is greater than the
                     // part 11 value (where supported)
-                    var partTwoValue = partTwoPacket.getSpnValue(spn.getId()).orElse(NOT_AVAILABLE);
-                    if (partTwoValue > spn.getValue()) {
-                        addFailure("6.11.13.4.d - Value received from " + module.getModuleName()
-                                           + " for " + spn
-                                           + " in part 2 was greater than part 11 value");
-                    }
+                    var partTwoValue = partTwoPacket.getSpnValue(spn.getId())
+                            .orElse(NOT_AVAILABLE);
 
-                    // 6.11.13.4.e.  Fail if active 100 hrs engine hours bin 1 SP 12389 < 600 seconds (where supported)
+                    // 6.11.13.4.e. Info if active 100 hrs engine hours bin 1 SP 12389 < 60 seconds (where supported)
                     if (spn.getId() == 12389) {
-                        if (spn.getValue() < 600) {
-                            addWarning("6.11.13.4.e - Active 100 hrs engine hours bin 1 SP " + spn.getId() + " value received is < 600 seconds from "
-                                    + module.getModuleName() + " for " + spn.getLabel());
+                        if (spn.getValue() < 60) {
+                            addInfo("6.11.13.4.e - Active 100 hrs engine hours bin 1 SP " + spn.getId()
+                                            + " value received is < 60 seconds from "
+                                            + module.getModuleName() + " for " + spn.getLabel());
                         }
                     }
 
                     // 6.11.13.4.f. Info, if stored 100 hrs engine hours > 0 seconds (where supported)
-                    if (pg == 64269 && spn.getValue() > 0) {
-                        addFailure("6.11.13.4.f - Active Tech stored engine hours received is > 0 seconds from "
-                                + module.getModuleName() + " for " + spn.getLabel());
+                    if (genericPacket.getPgnDefinition().getId() == 64269 && spn.getValue() > 0) {
```

Comment:
"(where supported)" indicate the value is expected to be NA (or null) in some cases.  We should check for null here.